docs/dev/developer-guides/transactions/blocks.md

```diff
@@ -58,11 +57,51 @@ To avoid this is why we built a purely fictional block containing the event o
 Block hashes in zkSync are deterministic and are derived from the following formula: "keccak256(l2_block_number)".
 The reason for having a deterministic block hash is that these hashes are not provable (remember that L2 blocks are not submitted to L1).
 Projects are advised not to use the L2 block hash as a source of randomness.
-
-### Block Properties
-
-- Timestamp: The current block's creation time in seconds that returns the timestamp of the L1 batch.
-- Block number: The unique sequential number for this block.
-- Gas limit: The current block gas limit, always returns `2^32-1`.
-- Coinbase: The current block miner’s address, returns the [bootloader](../system-contracts.md#bootloader) address.
-- Difficulty: The current block difficulty, returns `2500000000000000` (zkSync does not have proof of work consensus).
+## Block Properties
+
+## block.number and block.timestamp
+
+Depending on where it's used, if `block.number` and `block.timestamp` is retrieved via the `zksync-web3` API, mini blocks values are returned. It is a special construct on zkSync Era to make sure that the transactions are processed promptly. This approach is similar in other L2s, where each transaction belongs to a separate block, (which we call mini blocks), while these mini blocks are gathered together into batches that get sent to L1.
+
+In Solidity, `block.number` and `block.timestamp` returns the number and timestamp of the L1 batch respectively.
+### zksync-web3 API
+
+The following are the block properties returned when you use the `getBlock` method via the **zksync-web3** API.
+
+| Parameter     | Description                                                                                           |
+| ------------- | ------------------------------------------------------------------------------------------------------|
+| hash          | The hash of the block.                                                                                |
+| parentHash    | It refers to the hash of the parent block in L1.                                                      |
+| number        | The unique sequential number for this block.                                                          |
+| timestamp     | The current block's creation time in seconds returns the timestamp of the L2 block time.              |
```

Comment:
Maybe just leave "The current block's creation time in seconds"? Though I don't 100% remember whether it is in seconds or milliseconds. You can test it out locally